### ISSUE TITLE
'placeholder' attribute for core/list block in a block template

### ISSUE THREAD

**Issue by amarinediary** [WordPress/gutenberg#29782]
## What problem does this address?
Currently the core/list block doesn't have any editable visual cue beside the default "Write a list...". All other text based block have either a `content` or a `placeholder` attribute. 

The ability to set a placeholder for core/list blocks would enable a better onboarding for users in a block template based context.

## What is your proposed solution?
Adding a `placeholder` attribute to the core/list block.

**Comment by ntsekouras** [comment]
Hey 👋 - Do you mean showing the `Write list…` placeholder when inserting a new `List` block?

**Comment by amarinediary** [comment]
@ntsekouras Yes, exactly
